apps/api/configuration/saml/settings-staging.json

```diff
@@ -0,0 +1,60 @@
+{
+	"strict": true,
+	"debug": true,
+	"sp": {
+		"entityId": "https://staging.irvinehacks.com/shibboleth",
+		"assertionConsumerService": {
+			"url": "https://staging.irvinehacks.com/api/saml/acs",
+			"binding": "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST"
+		},
+		"singleLogoutService": {
+			"url": "https://staging.irvinehacks.com/api/saml/sls",
+			"binding": "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect"
+		},
+		"attributeConsumingService": {
+			"serviceName": "IrvineHacks Website",
+			"serviceDescription": "Website for IrvineHacks, Orange County's largest hackathon.",
+			"requestedAttributes": [
+				{
+					"nameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri",
+					"name": "urn:oid:0.9.2342.19200300.100.1.3",
+					"isRequired": true,
+					"friendlyName": "email"
+				},
+				{
+					"nameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri",
+					"name": "urn:oid:2.16.840.1.113730.3.1.241",
+					"isRequired": true,
+					"friendlyName": "displayName"
+				},
+				{
+					"nameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri",
+					"name": "urn:oid:2.16.840.1.113916.5.6.1.1",
+					"isRequired": true,
+					"friendlyName": "ucinetid"
+				},
+				{
+					"nameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri",
+					"name": "urn:oid:2.16.840.1.113916.5.6.1.59",
+					"isRequired": true,
+					"friendlyName": "uciaffiliation"
+				}
+			]
+		},
+		"NameIDFormat": "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified",
+		"x509cert": "",
+		"privateKey": ""
+	},
+	"idp": {
+		"entityId": "urn:mace:incommon:uci.edu",
```

Comment:
Note: this is not the correct entity ID for the QA endpoint, it should be `https://shib-qa.service.uci.edu/idp`